lib/check-pcb-components-outside-board/areComponentsOutOfBoard.ts

```diff
@@ -0,0 +1,46 @@
+import type {
```

Comment:
make sure the file name matches the function name, the convention on this repo is `check*`